web/src/components/FooterLinksList.tsx

```diff
@@ -0,0 +1,47 @@
+import ListItem from '@mui/material/ListItem'
+import ListItemButton from '@mui/material/ListItemButton'
+import ListItemText from '@mui/material/ListItemText'
+import Typography from '@mui/material/Typography'
+import { styled } from '@mui/material/styles'
+import React, { ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import Link from './base/Link'
+
+const StyledListItem = styled(ListItem)`
+  width: fit-content;
+
+  ${props => props.theme.breakpoints.down('md')} {
+    width: 100%;
+  }
+`
+
+export type FooterLinkItem = {
+  to: string
+  text: string
+  prefix?: string
+}
+
+type FooterLinksListProps = {
+  linkItems: FooterLinkItem[]
+}
+
+const FooterLinksList = ({ linkItems }: FooterLinksListProps): ReactElement[] => {
```

Comment:
🙃 Personally, I think its weird to have a component called `Lists` that only maps items without actually wrapping it in a list. I'd therefore prefer to just have a `FooterLink` component instead and do the mapping in the place where we actually use the items? This way, it would also be easier to apply styling with styled components and the `className` prop, e.g. to set the height of the buttons.
```suggestion
const FooterLink = ({ to, label }: FooterLinkProps): ReactElement => {
```